ansible/assets-upload.yml

```diff
@@ -18,18 +21,39 @@
         storage_account_name: "{{ azure_public_storage_account_name }}"
         storage_account_key: "{{ azure_public_storage_account_key }}"
         storage_account_sas_token: "{{ azure_public_storage_account_sas }}"
-      when: cloud_service_provider == "azure"
- 
+
     - name: delete files and folders from azure storage using azcopy
       include_role:
         name: azure-cloud-storage
         tasks_from: delete-using-azcopy.yml
-      when: cloud_service_provider == "azure"
 
     - name: upload batch of files to azure storage
       include_role:
         name: azure-cloud-storage
         tasks_from: blob-upload-batch.yml
       vars:
         local_file_or_folder_path: "{{ assets }}"
-      when: cloud_service_provider == "azure"
+    when: cloud_service_provider == "azure"
+
+  #GCP
+  - name: this block consists of tasks related to azure storage
+    block:
+    - name: set common gcloud variables
+      set_fact:
+        dest_folder_name: "{{ player_cdn_storage }}"
+        dest_folder_path: ""
+        file_delete_pattern: "{{ player_cdn_storage }}/"
+        gcp_bucket_name: "{{ gcloud_public_bucket_name }}"
```

Comment:
I don't see this added in private repo template. Please include them in your next PR.